### PR TITLE
Uppmax singularity config

### PR DESCRIPTION
**Pull request by remiolsen**
@senthil10

### CODE REVIEW

File: conf/any_slurm.config [hunk 1]
```diff
@@ -8,12 +8,7 @@ vim: syntax=groovy
  */
 
 
-if (params.clusterOptions == [:]){
-    throw new RuntimeException('You must specify --clusterOptions. For instance "-A <your_slurm_account>". Use an empty string if no options are applicable for your system.')
-}
-else {
-    process.clusterOptions = params.clusterOptions
-}
+process.clusterOptions = params.clusterOptions
```
Comment (senthil10) [inline conf/any_slurm.config:11]: Do we need this ? I guess not having this line won't make any difference :)

**Review by senthil10 — APPROVED** [review]
Just had one tiny comment, but happy to merge if you want it to be like that 👍

File: conf/any_slurm.config (hunk 1 above)
Comment (remiolsen) [reply conf/any_slurm.config:11]: I'm not sure 🤷‍♂️

Comment (senthil10) [reply conf/any_slurm.config:11]: hehe.. me neither. Anyway it doesn't hurt, so I will merge 👍

Comment (senthil10) [reply conf/any_slurm.config:11]: Oh it was merged already 😜

Comment (remiolsen) [reply conf/any_slurm.config:11]: 💨

**Pull request merged**